webapps-schema/src/main/java/io/camunda/webapps/schema/entities/operate/ErrorType.java

```diff
@@ -14,7 +14,8 @@ public enum ErrorType {
   UNSPECIFIED("Unspecified"),
   UNKNOWN("Unknown"),
   IO_MAPPING_ERROR("I/O mapping error"),
-  JOB_NO_RETRIES("No more retries left"),
+  JOB_NO_RETRIES("No more retries left", true),
+  EXECUTION_LISTENER_NO_RETRIES("No more retries left", true),
```

Comment:
@kristinkomschow one thought about this line, after I saw https://github.com/camunda/camunda/pull/24127: Should we have a different message here, e.g. `Execution Listener no more retries left`? I'm not sure what the significance of it is. Do we display it in the frontend?

Reply:
In the figma prototype there was no distinction between the message for Listeners or normal Jobs, but I find it logical to add "Execution Listener" to the message if we have a separate error type for it.
And yes, it gets displayed in the frontend as "Incident Type".

Reply:
I adjusted the wording both here and on the maintenance branch. I also rebased because there were concurrent changes in `ResolveIncidentHandler`. I will let the CI run and then merge it tomorrow.